src/hooks/makeQuery.ts

```diff
@@ -48,6 +48,37 @@ function makeQuery<TData, TVariables>(
       },
       errorPolicy: "all",
       fetchPolicy: "cache-and-network",
+      onError: async (error: ApolloError) => {
+        if (error.graphQLErrors.some(isJwtError)) {
```

Comment:
PR has this logic in 2 separate places. Can we handle error handling in one place? 

https://github.com/mirumee/saleor-dashboard/blob/7591dac2bc5a519e46563334b40348d77b533fa0/src/queries.tsx#L83

https://github.com/mirumee/saleor-dashboard/blob/7591dac2bc5a519e46563334b40348d77b533fa0/src/hooks/makeQuery.ts#L52